examples/envs/lung/lung_pid.ipynb

```diff
@@ -0,0 +1,182 @@
+{
+ "cells": [
```

Comment:
Can we also clean up the unused comments / lines of code? This is for public consumption, after all! You can also add commentary as appropriate.